mkdocs.yml

```diff
@@ -212,8 +212,8 @@ markdown_extensions:
     # pymdownx.superfences is deprecated.
     # It must be replaced with pymdownx.tabbed but it requires new CSS
     # TODO with material 5 update
-    - pymdownx.superfences:
-        legacy_tab_classes: true
+    - pymdownx.superfences
```

Comment:
I'm not sure what this change means.

Reply:
Publishing stuff

Reply:
It broke the tabs, so it was then reversed in another PR #368. This should not have been changed. If you have any doubt on things like this, ask.